interview-questions.md

```diff
@@ -7,49 +7,51 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Are Ruby methods accessors or mutators?
 
-  Your answer:
+  Your answer: Ruby methods can be both mutators and accessors.
 
-  Researched answer:
+  Researched answer:There are both types of methods in ruby. Mutators set the value of an attribute of an object while an accessor is a method that gets the value of the attribute.
 
 
 
 2. What is a block in Ruby?
 
-  Your answer:
+  Your answer: Block is the a piece of code that is enclosed with certain logic.
 
-  Researched answer:
+  Researched answer: A block is a chunk of code that will assign a name to a block. It is always enclosed within braces and blocks are invoke a from of function with the same name as that of the block.
```

Comment:
I would read through this it's pretty cool :) http://rubylearning.com/satishtalim/ruby_blocks.html